indexer/migrations/20230523_create_schema.sql

```diff
@@ -80,10 +86,42 @@ CREATE TABLE IF NOT EXISTS output_proposals (
     l1_contract_event_guid VARCHAR REFERENCES l1_contract_events(guid)
 );
 
+/**
+ * Derived data
+ * These schemas may change and need to resync
+ * We should consider syncing this from the CHAIN data instead of directly
+ * from the processor
+ */
+
 /**
  * BRIDGING DATA
  */
 
+ /**
+ * TOKEN DATA
+ */
+
+-- L1 Token table
+CREATE TABLE IF NOT EXISTS l1_tokens (
+    token_address   VARCHAR PRIMARY KEY,
+    name            VARCHAR NOT NULL,
+    symbol          VARCHAR NOT NULL,
+    decimals        INTEGER NOT NULL CHECK (decimals >= 0 AND decimals <= 18),
+    bridge_address  VARCHAR NOT NULL,
+    l2_token_address VARCHAR REFERENCES l2_tokens(token_address)
```

Comment:
Could make this not reference l2_tokens too not sure which way is better but 1 relationship is good enough. This seems more strict though so my guess is this is better?

Reply:
nvm it literally can't reference l2_tokens